### DISCUSSION TITLE
一直在想，v2ray如果可以支持serverless部署存活性会高很多

### DISCUSSION THREAD

**Discussion by tra4less** [v2fly/v2ray-core#1385] [Ideas]
如上

**Comment by junbao-zhou** [comment]
同感

**Reply by junbao-zhou** [reply]
可以节省很多资源，

**Comment by xiaokangwang** [comment]
请大概描述一项想支持哪个Serverless平台？

**Reply by tra4less** [reply]
serverless只能通过ws实现,每个平台的ws都要依赖apigateway,貌似没法一套代码通用。

**Reply by zizifn** [reply]
> serverless只能通过ws实现,每个平台的ws都要依赖apigateway,貌似没法一套代码通用。

不可以http2吗？

**Comment by Adkimsm** [comment]
heroku已经有相关仓库了，github一搜一大堆把

**Reply by Adkimsm** [reply]
vercel的话由于限制链接时间，应该是没法用的

**Reply by tra4less** [reply]
heroku是可以用的，但是它不是serverless

**Reply by Adkimsm** [reply]
啊着，Paas 不算 serverless吗

**Reply by tra4less** [reply]
特指Faas
